subworkflows/local/build_references.nf

```diff
@@ -142,7 +143,15 @@ workflow BUILD_REFERENCES {
             !file(params.starfusion_ref_stub_check).exists() || file(params.starfusion_ref_stub_check).isEmpty() )) {
             STARFUSION_BUILD(ch_fasta, ch_gtf, params.fusion_annot_lib, params.species)
             ch_versions = ch_versions.mix(STARFUSION_BUILD.out.versions)
-            ch_starfusion_ref = STARFUSION_BUILD.out.reference
+            if (params.ctatsplicing || params.all) {
+                CTATSPLICING_PREPGENOMELIB(
+                    STARFUSION_BUILD.out.reference
+                )
+                ch_versions = ch_versions.mix(CTATSPLICING_PREPGENOMELIB.out.versions)
+                ch_starfusion_ref = CTATSPLICING_PREPGENOMELIB.out.reference
```

Comment:
Replacing the reference?

Reply:
I find it weird to first build the starfusion refs if one replaces them after, or is it just an addition to the starfusion references? In this case the logic needs to be reworked

Reply:
No it doesn't replace the reference, but updates it. It adds a couple of files to it

Reply:
I see now how it works, nice!